styles/styles.css

```
@@ -4,5 +4,126 @@
 }
 
 body {
-   font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; 
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.container {
+    padding: 16px;
+}
+
+.imgcontainer {
```

use the same naming convention everywhere

```suggestion
.img-container {
```